.github/scripts/generate_ci_workflows.py

```diff
@@ -157,7 +157,7 @@ def generate_workflow_file(self, workflow_template: jinja2.Template) -> None:
     ),
     CIWorkflow(
         arch="windows",
-        build_environment="win-vs2019-cuda10-cudnn7-py3",
+        build_environment="win-vs2019-cuda10-py3",
```

Comment:
can you add the minor version for cuda here

Reply:
```suggestion
        build_environment="win-vs2019-cuda10.1-py3",
```

Reply:
actually this doesn't work. need to regenerate the workflows. let me actually do it instead of being lazy